test/util/fonts/getTextByFontProperties.js

```diff
@@ -481,6 +481,24 @@ describe('lib/util/fonts/getTextByFontProperties', function () {
             ]);
         });
 
+        it('should support :after without content', function () {
+            var htmlText = [
+                '<style>h1:after { font-family: font1 !important; }</style>',
+                '<h1></h1>'
+            ].join('\n');
+
+            return expect(htmlText, 'to exhaustively satisfy computed font properties', [
+                {
+                    text: '',
+                    props: {
+                        'font-family': 'font1',
+                        'font-weight': 700,
+                        'font-style': 'normal'
+                    }
+                }
+            ]);
```

Comment:
I would expect the empty array here, since pseudo elements don't really exist without `content`